test/validation/api/classification/odp_classification_basic.c

```diff
@@ -10,23 +11,29 @@
 
 #define PMR_SET_NUM	5
 
-static void classification_test_default_values(void)
+static void test_defaults(uint8_t fill_byte)
```

Comment:
Type is already a byte so name could be just `fill`.

Reply:
Renamed in v2.